src/scripts/main.js

```diff
@@ -1 +1,10 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+    resolve(`First promise was resolved`);
+    setTimeout(() => {
+        reject(new Error(`First promise was rejected`))
+    },3000)
+})
```

Comment:
Once a promise is resolved, it cannot be rejected or resolved again. The rejection inside the setTimeout will have no effect because the promise is already resolved. If you want to handle both resolve and reject scenarios, consider restructuring the logic.